libraries/AC_Avoidance/AP_OADatabase.h

```diff
@@ -104,6 +116,11 @@ class AP_OADatabase {
         uint16_t        size;                               // cached value of _database_size_param that sticks after initialized
     } _database;
 
+    struct {
+        OA_DBItem_Tracked       *items;                             // array of tracked objects in the database
```

Comment:
a little bit of extra white space maybe before the comment.. if possible put a comment above the structure.